### ISSUE TITLE
Spring boot and apache spark integration issue

### ISSUE THREAD

**Issue by pretrainedjon** [bitnami/containers#43813]
### Name and Version

bitnami/spark:3.3.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. In docker we setup docker container and launched
2. in config file we created spark session, SparkConf.setName("aaa).setMaster(master ip and port)//local ip of pc which is working docker. here connection cannot established spring boot application and spark master. is it possible to connect remote connection which working on docker

### What do you see instead?

how can configure spark in order to connect spring boot with apache spark?

**Comment by carrodher** [comment]
It seems it is not an issue related to the Bitnami container image or Helm chart but about how the application or environment is being used/configured.

For information regarding the application itself, customization of the content within the application, or questions about the use of the technology or infrastructure; we highly recommend checking forums and user guides made available by the project behind the application or the technology.

That said, we will keep this ticket open until the stale bot closes it just in case someone from the community adds some valuable info.

**Comment by pretrainedjon** [comment]
I connected to the master on docker , I mounted the volume like this /Users/data:/home/data. then I checked via docker desktop and I can see files which I have mounted.  But when I try to read with path:"/home/data/test.csv" , exception occured [PATH_NOT_FOUND] Analyze exception. the second problem is, I tried to connect to mongodb which is installed in docker, It can connect db, show schema. but I want to use show 5 rows of the collection. It throws exception: cannot assign instance of scala.collection.immutable.List$SerializationProxy to field org.apache.spark.sql.execution.datasources.v2.DataSourceRDDPartition.inputPartitions of type scala.collection.Seq in instance of org.apache.spark.sql.execution.datasources.v2.DataSourceRDDPartition
